<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Duplicates</class>
 <widget class="QMainWindow" name="Duplicates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find duplicate folders</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/Duplicates/folder.svg</normaloff>:/Duplicates/folder.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_folder">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_folderPick">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pick folder...</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources/resources.qrc">
          <normaloff>:/Duplicates/folder.svg</normaloff>:/Duplicates/folder.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_search">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Search for duplicates</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources/resources.qrc">
          <normaloff>:/Duplicates/magnifying-glass.svg</normaloff>:/Duplicates/magnifying-glass.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="m_progress">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Duplicates info table</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="m_table">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Folder</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Parent 1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size 1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Parent 2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size 2</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Inserted:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_inserted">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Collisions:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_collisions">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_about">
        <property name="toolTip">
         <string>About the application...</string>
        </property>
        <property name="text">
         <string>About...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_quit">
        <property name="toolTip">
         <string>Exits the application.</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
